samples/apps/copilot-chat-app/webapi/Model/PluginAuthHeaders.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace SemanticKernel.Service.Plugins;
+
+public class PluginAuthHeaders
+{
+    [FromHeader(Name = "x-sk-copilot-graph-authorization")]
+    public string? GraphAuthentication { get; set; }
+
+    [FromHeader(Name = "x-sk-copilot-jira-authorization")]
+    public string? JiraAuthentication { get; set; }
+
+    [FromHeader(Name = "x-sk-copilot-github-authorization")]
```

Comment:
Not critical for now, but consider having a generic way to extract auth headers, like adding all headers that follow the "x-sk-copilot-XXX-authorization" format to a dictionary.  This way, when we get new plug-ins, we don't have to modify this object. This would involved the creation of a method to be invoked in the relevant controllers.

Reply:
For now, it's a better to keep the PluginAuthHeaders class to keep all the plug-ins well defined. I think your proposal is great for discovery, but we still need to know exactly what plug-ins to expect in order to register them properly since each can require different Authentication methods and we have to instantiate the right one.  i.e., Jira uses basic and Graph uses Bearer 

Not priority right now, will note to revisit in the future (i.e., have a header send in which skills need to be registered to run the query)